Makefile

```diff
@@ -78,6 +78,8 @@ build/llvm.BUILT:
 		-DCLANG_LINKS_TO_CREATE="$(call join-with,;,$(CLANG_LINKS_TO_CREATE))" \
 		-DCMAKE_BUILD_TYPE=MinSizeRel \
 		-DLLVM_ENABLE_TERMINFO=OFF \
+		-DLLVM_LINK_LLVM_DYLIB=ON \
+		-DLLVM_VERSION_SUFFIX=-wasi-sdk \
```

Comment:
Does this mean that `bin/clang-18` is now `bin/clang-18-wasi-sdk`?    Or does it just apply the `.so` file?

Reply:
Looks like it's just the `*.so` files, from the installation folder I get:

```
 $ fd -- -wasi-sdk
lib/libLLVM-18-wasi-sdk.so
lib/libLLVM.so.18.1-wasi-sdk
lib/libclang-cpp.so.18.1-wasi-sdk
```

can confirm that `bin/clang-18` is still named just that, no `wasi-sdk`